androidPackageTut/package.json

```diff
@@ -1,22 +1,22 @@
 {
-	"name": "androidPackageTut",
-	"version": "0.0.1",
-	"private": true,
-	"scripts": {
-		"start": "node node_modules/react-native/local-cli/cli.js start",
-		"test": "jest"
-	},
-	"dependencies": {
-		"react": "15.4.1",
-		"react-native": "0.39.2"
-	},
-	"devDependencies": {
-		"babel-jest": "18.0.0",
-		"babel-preset-react-native": "1.9.1",
-		"jest": "18.1.0",
-		"react-test-renderer": "15.4.1"
-	},
-	"jest": {
-		"preset": "react-native"
-	}
-}
\ No newline at end of file
+  "name": "androidPackageTut",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "node node_modules/react-native/local-cli/cli.js start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "react": "15.1.0",
+    "react-native": "0.29.1"
```

Comment:
Your tutorial uses an old version of RN which uses the old way of creating native modules. Ensure that the version your using is the same as the one in the tutorial